config/packages/security.yaml

```diff
@@ -13,6 +13,8 @@ security:
             entity:
                 class: App\Entity\User
                 property: email
+                manager_name: goteo
```

Comment:
This is required, otherwise the `UserProvider` in `Authenticators` will try to use the default connection (where only OAuth tables and entities are)